README.md

```diff
@@ -42,13 +42,13 @@ See [installation](https://www.cursorless.org/docs/user/installation/) for insta
 This extension contributes the following settings:
 
 - `cursorless.showOnStart`: Whether decorations should appear on workspace start
-- `cursorless.hatSizeAdjustment`: Percentage to increase or decrease hat size; positive increases size
-- `cursorless.hatVerticalOffset`: How much to vertically shift the hats as a percentage of font size; positive is up
-- `cursorless.hatEnablement.colors`: Whether to enable particular hat colors.
-- `cursorless.hatEnablement.shapes`: Whether to enable particular hat shapes.
-- `cursorless.hatPenalties.colors`: How much to penalize each hat color. You will probably want to set this one to the number of syllables in the given style. Cursorless will then sort every style combination by number of syllables to refer to it.
-- `cursorless.hatPenalties.shapes`: How much to penalize each hat shape. You will probably want to set this one to the number of syllables in the given style. Cursorless will then sort every style combination by number of syllables to refer to it.
-- `cursorless.maximumHatStylePenalty`: The maximum allowed penalty for a hat style. Any hat style whose penalty is greater than this amount will not be used. A hat style penalty is defined to be the shape penalty plus the colour penalty. Setting this value less than or equal to zero is treated as no maximum.
+- `cursorless.hatSettings.hatSizeAdjustment`: Percentage to increase or decrease hat size; positive increases size
+- `cursorless.hatSettings.hatVerticalOffset`: How much to vertically shift the hats as a percentage of font size; positive is up
+- `cursorless.hatSettings.hatEnablement.colors`: Whether to enable particular hat colors.
+- `cursorless.hatSettings.hatEnablement.shapes`: Whether to enable particular hat shapes.
+- `cursorless.hatSettings.hatPenalties.colors`: How much to penalize each hat color. You will probably want to set this one to the number of syllables in the given style. Cursorless will then sort every style combination by number of syllables to refer to it.
+- `cursorless.hatSettings.hatPenalties.shapes`: How much to penalize each hat shape. You will probably want to set this one to the number of syllables in the given style. Cursorless will then sort every style combination by number of syllables to refer to it.
+- `cursorless.hatSettings.maximumHatStylePenalty`: The maximum allowed penalty for a hat style. Any hat style whose penalty is greater than this amount will not be used. A hat style penalty is defined to be the shape penalty plus the colour penalty. Setting this value less than or equal to zero is treated as no maximum.
```

Comment:
If these names change then we need some migration code for those that have modified these settings. I don't see any.